.github/workflows/pull-request.yaml

```diff
@@ -5,16 +5,19 @@ on:
     branches:
       - main
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
 jobs:
 
   check-code-quality:
     name: Run charm code quality checks
     uses: ./.github/workflows/_quality_check.yaml
     secrets: inherit
     with:
-      tox-version: ""
+      tox-version: ""  # latest tox
```

Comment:
not-blocking: the default value should be changed in the bootstack-actions, and not here.

Reply:
Yeah, we need to update `func.yaml@v3`. Resolving this for now, will open another PR later when `func.yaml@v3` is fixed

Reply:
It was fixed in the all version of bootstack-actions.